test/e2e/specs/all-features/record/related-table-link.spec.js

```diff
@@ -126,7 +126,8 @@ describe('View existing record,', function() {
                     browser.wait(EC.elementToBeClickable(addRelatedRecordLink), browser.params.defaultTimeout);
 
                     expect(addRelatedRecordLink.isDisplayed()).toBeTruthy();
-                    addRelatedRecordLink.click().then(function() {
+                        browser.executeScript('window.scrollTo(0,200);').then(function() {
```

Comment:
This is an interesting way to manage the browser not seeing the element in the DOM. I like it.